src/legacy/core_plugins/data/public/search/search_service.ts

```diff
@@ -33,3 +43,5 @@ export class SearchService {
 /** @public */
 
 export type SearchSetup = ReturnType<SearchService['setup']>;
+
+export { SavedQuery, SavedQueryAttributes } from './search_bar';
```

Comment:
nit: These are already exported from `index.ts`

Reply:
But `index.ts` is importing these from `search_service.ts`. I'm happy to change it so `index.ts` exports the definitions directly if you think that's better, I think we were just following an existing pattern from one of the other services.

Reply:
It seems like `data/index.ts` is importing 

```
export { SearchBar, SearchBarProps, SavedQueryAttributes, SavedQuery } from './search';
```
So it seems to me they are not needed here.

Reply:
👍 ba733b2